ci_build_images/msan.fragment.Dockerfile

```diff
@@ -93,51 +93,10 @@ RUN . /etc/os-release \
 COPY --from=rr /tmp/install/usr/ /usr/
 
 # ASAN/UBSAN
+COPY msan.motd /etc/motd
+
 RUN echo "cat /etc/motd" > ~buildbot/.bashrc ; \
-    printf "\
-This is a container for ASAN, UBSAN and MSAN building\n\
-\n\
-A basic MSAN build can be achieved with\n\
-\n\
-cmake -DWITH_EMBEDDED_SERVER=OFF \\ \n\
-      -DWITH_INNODB_{BZIP2,LZ4,LZMA,LZO,SNAPPY}=OFF \\ \n\
-      -DPLUGIN_{MROONGA,ROCKSDB,OQGRAPH,SPIDER}=NO  \\ \n\
-      -DWITH_ZLIB=bundled  \\ \n\
-      -DHAVE_LIBAIO_H=0    \\ \n\
-      -DCMAKE_DISABLE_FIND_PACKAGE_{URING,LIBAIO}=1  \\ \n\
```

Comment:
Maybe the line about disabling `liburing` needs to remain in `msan.motd` until the library has been made MSAN friendly. The interface to `libaio` was indeed fixed in MariaDB/server#3949 a while ago.

Reply:
Sure - readding disable uring.

Fixed liburing properly this time - https://github.com/MariaDB/server/pull/4257

both libaio and liburinga are in the container.